src/main/java/endpointurl/Endpoint.java

```diff
@@ -9,8 +9,8 @@
     public static final String ORDER = ORDERS + "/{" + ElementParam.ID + "}";
     public static final String CONTACTS = "/" + Element.CONTACT;
     public static final String CONTACT = CONTACTS + "/{" + ElementParam.ID + "}";
-    public static final String CONTRACTS = "/" + Element.CONTRACT;
-    public static final String CONTRACT = CONTRACTS + "/{" + ElementParam.ID + "}";
+    public static final String CAMPAIGNS = "/" + Element.CAMPAIGN;
```

Comment:
Using class attributes to build a String that should represent only an endpoint may add complexity to the class, could probably be left as String since they are already final.